src/Tests/dotnet-workload-list.Tests/GivenAnMsiInstallation.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.DotNet.Workloads.Workload.Install.InstallRecord;
+using Microsoft.NET.TestFramework;
+using Microsoft.Win32;
+using Xunit;
+
+namespace Microsoft.DotNet.Cli.Workload.List.Tests
+{
+    [Collection("MsiWorkloadRecords")]
+    public class GivenAnMsiInstallation
+    {
+        // Override HKLM to HKCU so we can run tests without needing elevation
+        private static MsiWorkloadInstallationRecordManager RecordManager = new MsiWorkloadInstallationRecordManager(
+            Registry.CurrentUser,
+            @"SOFTWARE\Microsoft\dotnet-test\InstalledWorkloads\Standalone");
+
+        [WindowsOnlyFact]
+        public void ItCanReadWorkloadRecords()
```

Comment:
nit: We've been using the `Given... it...` test name pattern for all our workload tests so far

Reply:
I'll fix this up